pkg/iac/scanners/terraform/parser/resolvers/registry_test.go

```diff
@@ -0,0 +1,50 @@
+package resolvers
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_getPrivateRegistryTokenFromEnvVars(t *testing.T) {
```

Comment:
How about using a table test here? Here's an example https://github.com/aquasecurity/trivy/blob/main/pkg/licensing/classifier_test.go#L15

Reply:
Done, see what you think